app/scripts/directives/schoollist.coffee

```diff
@@ -18,6 +18,8 @@ angular.module 'edudashApp'
       hover: '=hover'
       unHover: '=unHover'
       property: '@property'
+      showModal: '=showmodal'
+      showTotalModal: '@showtotalmodal'
```

Comment:
I have include the link to open the modal in the schoollist, so that is why we need the size of the modal